content/operator/pkp/index.de.md

```diff
@@ -0,0 +1,257 @@
+- Reservierungspflicht für EIP und EIC sowie teilweise in IC und TLK Züge
+- Gültigkeit der PKP FIP-Tickets auch in S-Bahn Systemen von Warschau, Danzig/Gdingen und Masowien sowie einigen Strecken der Koleje Dolnośląskie (KD)
+
+## Gültigkeit FIP-Tickets
+
+FIP Freifahrtsschein: ✅ \
+FIP Freifahrt Angehörige: ✅ \
+FIP 50 Tickets: ✅
+
+FIP Freifahrtscheine und FIP 50 Tickets sind bei PKP gültig. Dazu gehören auch:
+- PKP InterCity
+- POLREGIO S.A
+- Koleje Mazowieckie (KM) (Regionalzüge um Warschau bzw. der Region Masowien)
+- PKP Szybka Kolej Miejska w Trojmiescie (SKM) (S-Bahnen zwischen Danzig, Gdingen und Sopot)
+- Warszawska Kolej Dojazdowa (KD) (S-Bahnen im Großraum Warschau)
+- Einige Strecken der Koleje Dolnośląskie (KD). [Mehr Informationen]({{< ref "/operator/kd" >}} "KD")
+
+Bei grenzüberschreitenden Fahrten muss entweder ein durchgängiges FIP 50 Ticket oder FIP Freifahrtscheine beider Länder vorhanden sein.
+
+## Zugkategorien und Reservierungen
+
+{{% highlight important %}}
+Polnische Fernzüge sind teilweise reservierungspflichtig. Die Reservierungspflicht gilt für die Züge für alle EIP und EIC Züge sowie teilweise für IC und TLK Züge.
+{{% /highlight %}}
+
+{{% expander "Express Intercity Premium (EIP) ⚠️" traincategory category %}}
+**Beschreibung:** \
+Hochgeschwindigkeitszug mit Neigetechnik (Pendolino) betrieben von PKP Intercity, die Gdynia/Gdańsk und Kraków/Katowice über Warschau fahren. \
+In der 1. Klasse sind Speisen und Getränke am Platz im Ticketpreis inkludiert. [^2] \
+Die Züge sind aufpreispflichtig mit einem erhöten Reservierungsentgelt. Tickets und Reservierungen können nicht an Bord des Zuges gekauft werden!
+
+Bei Reservierungen die nicht zu einem entsprechenden Preis durch die PKP verkauft wurden, kann die Zahlung eines Zuschlags an Bord des Zuges erforderlich sein. [^1] \
+**Reservierung möglich:** ja \
+**Reservierungspflicht:** ⚠️ ja \
+**Kosten für Reservierung:** \
+43 PLN \
+(Zum Festpreis und nur über die PKP buchbar)
+{{% /expander %}}
+
+{{% expander "Express InterCity (EIC) ⚠️" traincategory category %}}
+**Beschreibung:** \
+Komfortable Züge betrieben von PKP Intercity mit modernisierten Reisezugwagen mit Unterwegshalt nur in den wichtigsten Städten. Grenzüberschreitende EICs werden in anderen Ländern auch als EC bezeichnet. [^3] \
+In der 1. und 2. Klasse gibt es ein kostenloses Wasser pro Person. \
+Die Züge sind immer reservierungspflichtig. \
+**Reservierung möglich:** ja \
+**Reservierungspflicht:** ⚠️ ja
+{{% /expander %}}
+
+
+{{% expander "InterCity (IC) ⚠️" traincategory category %}}
+**Beschreibung:** \
+Schnellzüge mit Reisezugwagen oder modernen Triebzügen betrieben von PKP Intercity mit Halt an wichtigen Bahnhöfen. [^4] \
+Einige Züge sind Reservierungspflichtig. Die Reservierungspflicht kann in der [Reiseauskunft der PKP](https://www.intercity.pl/en/) geprüft werden. \
+**Reservierung möglich:** ja \
+**Reservierungspflicht:** ⚠️ teilweise
```

Comment:
Wir sollten nochmal verifizieren, welche Zugkategorien reservierungspflichtig sind. Bei der PKP FIP steht das: https://www.intercity.pl/en/dokumenty/Interrail/[FIP_w_PKP_IC_eng_0807_2025.pdf

hier steht zum TLK aber was anderes:
https://www.intercity.pl/en/site/for-passengers/offers/oferta-interrail/rezerwacje-miejsc-kompendium-wiedzy.html

Irgendwie ist die PKP da in sich schon widersprüchlich...